### PR TITLE
German Translations

### PR DESCRIPTION
**Pull request by LadyViktoria**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -291,4 +291,27 @@
     <string name="primefill">Füllen</string>
     <string name="uploading">Hochladen</string>
     <string name="es_lang">Spanish</string>
+    <string name="basal_rate">Basalrate:</string>
+    <string name="base_profile_label">Basisprofil:</string>
+    <string name="batterydischarged">Pumpen Batterie entladen</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: "Pumpenbatterie"

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -291,4 +291,27 @@
...
+    <string name="androidaps_tempbasalendnote">Basal Temp Ende</string>
+    <string name="androidaps_tempbasalstartnote"></string>
+    <string name="danar_historyreload">neu laden</string>
+    <string name="high_mark">Hoch Markierung</string>
+    <string name="low_mark">Niedrig Markierung</string>
+    <string name="lowbattery">Niedrige Batterie</string>
+    <string name="pumpbusy">Pumpe ist beschäftigt</string>
+    <string name="pumperror">Pumpen Fehler</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: "Pumpenfehler"

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -291,4 +291,27 @@
...
+    <string name="pumpshutdown"></string>
+    <string name="send_to_pump">an Pumpe senden</string>
+    <string name="target_range">Zielbereich:</string>
+    <string name="units">Einheiten:</string>
+    <string name="wrongpumpdriverselected">Falscher Pumpentreiber ausgewählt</string>
+    <string name="old_data">veraltete Daten</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: In english this is all caps.
"VERALTETE DATEN" might make the whole line too long for the notification.
How about just "VERALTET"?

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (LadyViktoria) [inline app/src/main/res/values-de/strings.xml]: dürfte man so oder so schreiben. dachte auseinander ist leichter lesbar

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -291,4 +291,27 @@
...
+    <string name="batterydischarged">Pumpen Batterie entladen</string>
+    <string name="androidaps_tempbasalendnote">Basal Temp Ende</string>
+    <string name="androidaps_tempbasalstartnote"></string>
+    <string name="danar_historyreload">neu laden</string>
+    <string name="high_mark">Hoch Markierung</string>
+    <string name="low_mark">Niedrig Markierung</string>
+    <string name="lowbattery">Niedrige Batterie</string>
+    <string name="pumpbusy">Pumpe ist beschäftigt</string>
+    <string name="pumperror">Pumpen Fehler</string>
+    <string name="pumpshutdown"></string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: Here the translation seems to be missing.

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -291,4 +291,27 @@
     <string name="primefill">Füllen</string>
     <string name="uploading">Hochladen</string>
     <string name="es_lang">Spanish</string>
+    <string name="basal_rate">Basalrate:</string>
+    <string name="base_profile_label">Basisprofil:</string>
+    <string name="batterydischarged">Pumpen Batterie entladen</string>
+    <string name="androidaps_tempbasalendnote">Basal Temp Ende</string>
+    <string name="androidaps_tempbasalstartnote"></string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: Here the translation seems to be missing.

File: app/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -228,7 +228,7 @@
     <string name="danar_invalidinput">ungültige Eingabe</string>
     <string name="danar_valuenotsetproperly">Wert nicht korrekt gesetzt</string>
     <string name="devicenotfound">ausgewähltes Gerät nicht gefunden</string>
-    <string name="dismiss">ABWEISEN</string>
+    <string name="dismiss">VERWERFEN</string>
     <string name="enacted">Verordnet</string>
     <string name="end_user_license_agreement">Einbenutzervereinbarung</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: While you're at it @LadyViktoria, could you also please change this to "*End*benutzer...". thanks

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: Im Deutschen nicht. Ein mögliche Alternative nach neuer Rechtschreibung wäre "Pumpen-Fehler" mit Bindestrich.

**Comment by MilosKozak** [comment]
done? :)

**Comment by LadyViktoria** [comment]
i am confused now there are 2 commits :-(
how can i delte the other one an edit the first one

**Comment by MilosKozak** [comment]
final diff is ok? https://github.com/MilosKozak/AndroidAPS/pull/106/files

**Comment by LadyViktoria** [comment]
looking tomorrow again... closing now..

**Pull request closed without merging**